README.md

```diff
@@ -1,5 +1,6 @@
 # Readit - Command Line Tool
 
+[![Python](https://img.shields.io/badge/python-2.7%2C%203.6-blue.svg)]() [![LICENSE](https://img.shields.io/badge/license-GPLv3-yellow.svg?maxAge=2592000)]() [![Build Status](https://travis-ci.org/freeCodeCamp/how-to-contribute-to-open-source.svg?branch=master)](https://travis-ci.org/freeCodeCamp/how-to-contribute-to-open-source)
```

Comment:
Link for the Travis CI badge is of `freeCodeCamp` not `readit`
Click the badge form this page, https://travis-ci.org/ganeshhubale/readit and select Markdown 
Add that content here

Reply:
@bhavin192 
i am taking it from http://shields.io/ . I want to know both are same or  not.